docs/EN/Configuration/BG-Source.md

```diff
@@ -1,12 +1,20 @@
 # BG source
 
 **For users of Dexcom:**<Br>
-_If using G5 with xdrip+_<br>
-* If not already set up then download [xdrip](https://github.com/NightscoutFoundation/xDrip) and follow instructions on nightscout ([G4 without share](http://www.nightscout.info/wiki/welcome/nightscout-with-xdrip-wireless-bridge), [G4 share](http://www.nightscout.info/wiki/welcome/nightscout-with-xdrip-and-dexcom-share-wireless), [G5](http://www.nightscout.info/wiki/welcome/nightscout-with-xdrip-and-dexcom-share-wireless/xdrip-with-g5-support)).
+_If using G5 or G6 with xdrip+_<br>
+* If not already set up then download [xdrip](https://github.com/NightscoutFoundation/xDrip) and follow instructions on nightscout and xdrip FB group ([G4 without share](http://www.nightscout.info/wiki/welcome/nightscout-with-xdrip-wireless-bridge), [G4 share](http://www.nightscout.info/wiki/welcome/nightscout-with-xdrip-and-dexcom-share-wireless), [G5](http://www.nightscout.info/wiki/welcome/nightscout-with-xdrip-and-dexcom-share-wireless/xdrip-with-g5-support), [G6](https://www.facebook.com/groups/xDripG5/permalink/2380684062005075/)).
 * In xdrip go to Settings > Interapp Compatibility > Broadcast Data Locally and select ON.
-* In xdrip go to Settings > Interapp Compatibility > Accept Treatments and select OFF.
+* In xdrip go to Settings > Interapp Compatibility > Accept Treatments and select ON.
```

Comment:
Accept Treatments should be OFF

Reply:
In the setup proposed by me, xdrip is used only to upload glucose values to NS. This is eliminated by Settings > Cloud Upload > Nightscout Sync (REST API) >  Extra Options > Upload treatments and untick the checkbox .  Treatments are not uploaded.  This way we can accept treatments from AAPS in xdrip. The problem with duplicated treatments doesn't exist.